.taskcluster.yml

```diff
@@ -156,8 +156,8 @@ tasks:
                 routes:
                     $flattenDeep:
                         - checks
-                        - $if: 'level == "3"'
```

Comment:
I'm a bit concerned about this. For `cron` and `action`, the "project" value comes from build-decision, and we don't set a separate trust project for `staging-mozilla-vpn-client:
https://hg.mozilla.org/ci/ci-configuration/file/ee2e4aa3687f0de4eedd617ecca25494eca720bc/projects.yml#l826

I haven't looked into how build-decision is generating that value, but "project" is the only thing that would differentiate a bunch of the index routes below, e.g:
```
index.mozillavpn.v2.${project}.branch.${head_branch}.latest.taskgraph.decision-${cron.job_name}
```

Reply:
I'm hoping that project will come from the alias in `projects.yml`, in which case we should be fine.